src/data.ts

```diff
@@ -0,0 +1,34 @@
+import Network from '@big-whale-labs/stores/dist/models/Network'
+import env from 'helpers/env'
+
+export enum BadgeSourceType {
+  Email = 'Email',
+  ERC721 = 'ERC721',
+}
+
+export default {
+  ExternalERC721: {
+    ledger: env.VITE_SC_EXTERNAL_ERC721_LEDGER_CONTRACT_ADDRESS,
+    title: 'Mainnet NFT derivatives',
+    source: {
```

Comment:
Why do you need to nest `type` and `network` in `source`? If there's no reason, we shouldn't nest it

Reply:
As I can see, there's no reason, so let's remove it and use directly from the object